archive/2025-10-22-updates-mitre-atlastm.md

```diff
@@ -0,0 +1,81 @@
+---
+date: '2025-10-22'
+description: The September 2025 update of the ATLAS framework introduces version 4.6.0,
+  emphasizing Agentic AI TTPs in collaboration with Zenity. The data upgrade to v5.0.0
+  includes a new "Technique Maturity" field, evaluating techniques on feasibility,
+  demonstration, and real-world application. Notable additions include techniques
+  for AI agent context poisoning, credential harvesting, and data exfiltration via
+  AI tools. The release enhances the repository with practical insights, including
+  a new case study on circumventing ML phishing detection. This development reflects
+  a growing focus on securing AI-driven environments amid evolving threats.
+link: https://atlas.mitre.org/resources/updates/2025-09
+tags:
+- cybersecurity
+- case studies
+- TTPs
+- data analysis
+- AI
+title: Updates ◆ MITRE ATLAS™
+---
+
+- [Home](https://atlas.mitre.org/)
+- [resources](https://atlas.mitre.org/resources/info)
+- [updates](https://atlas.mitre.org/resources/updates)
+- 2025-09
+
+Updates
+
+All Updates
+
+[Oct 2025](https://atlas.mitre.org/resources/updates/2025-10) [Sep 2025](https://atlas.mitre.org/resources/updates/2025-09) [Aug 2025](https://atlas.mitre.org/resources/updates/2025-08) [Apr 2025](https://atlas.mitre.org/resources/updates/2025-04) [Mar 2025](https://atlas.mitre.org/resources/updates/2025-03) [Oct 2024](https://atlas.mitre.org/resources/updates/2024-10) [Mar 2024](https://atlas.mitre.org/resources/updates/2024-03) [Jan 2024](https://atlas.mitre.org/resources/updates/2024-01) [Oct 2023](https://atlas.mitre.org/resources/updates/2023-10) [Apr 2023](https://atlas.mitre.org/resources/updates/2023-04) [Feb 2023](https://atlas.mitre.org/resources/updates/2023-02) [Jan 2023](https://atlas.mitre.org/resources/updates/2023-01) [Oct 2022](https://atlas.mitre.org/resources/updates/2022-10) [Jul 2022](https://atlas.mitre.org/resources/updates/2022-07) [Mar 2022](https://atlas.mitre.org/resources/updates/2022-03) [Oct 2021](https://atlas.mitre.org/resources/updates/2021-10) [Jul 2021](https://atlas.mitre.org/resources/updates/2021-07) [Jun 2021](https://atlas.mitre.org/resources/updates/2021-06)
+
+#### September 2025
+
+##### Website v4.6.0
+
+This website update includes new TTPs focused on Agentic AI developed in collaboration with Zenity and a new case study from a user submissions.
```

Comment:
Really excited about this Zenity-MITRA collab! The Atlas team has been incredible to work with.